packages/grafana-api-clients/package.json

```diff
@@ -23,49 +23,57 @@
       "import": "./src/index.ts",
       "require": "./src/index.ts"
     },
-    "./advisor/v0alpha1": {
-      "import": "./src/clients/advisor/v0alpha1/index.ts",
-      "require": "./src/clients/advisor/v0alpha1/index.ts"
+    "./rtkq": {
+      "import": "./src/rtkq.ts",
+      "require": "./src/rtkq.ts"
     },
-    "./folder/v1beta1": {
-      "import": "./src/clients/folder/v1beta1/index.ts",
-      "require": "./src/clients/folder/v1beta1/index.ts"
+    "./rtkq/advisor/v0alpha1": {
```

Comment:
I am wondering if consumers need to know that it's an rtk client when importing this package, e.g., cild we omit this for exports?